content/blog/2023-08-31-localstack-has-joined-aws-marketplace-to-streamline-local-cloud-development-testing/index.md

```diff
@@ -0,0 +1,52 @@
+
+The LocalStack Team edition delivers a range of features aimed at enhancing your cloud productivity while seamlessly aligning with your existing workflows. These features are designed to reduce friction and adapt as your infrastructure & workflows scale, with:
+
+- **Cloud Pods**: Store a persistent snapshot of your LocalStack instance in the Web Application and inject Cloud Pods from your team members into your LocalStack instance for collaborative debugging.
+- **Stack Insights**: Access AWS usage telemetry, including service invocations, the user agent and the particular service during a specific LocalStack spin-up to enable deep debugging within your team.
+- **LocalStack in CI**: Spin up LocalStack in your CI pipelines to run tests before pushing out changes to production or to create ephemeral environments to showcase your changes with LocalStack.
+- **LocalStack Extensions**: Extend and customise LocalStack to integrate custom service emulators, such as Cloudflare & Stripe, into your LocalStack instance to enable your team go beyond AWS.
+
+With LocalStack now available in the AWS Marketplace, you can quickly leverage a streamlined procurement to access our cloud development framework. 
+
+## Subscribing to LocalStack in AWS Marketplace
+
+To launch LocalStack with AWS Marketplace, you can follow the steps below:
+
+- Go to **AWS Marketplace Subscriptions** in the AWS Console and locate **LocalStack** under **Discover Products**.
+- Review the terms, select the desired number of seats, contract renewal strategy, and accept the contract. Accepting the contract will initiate a Marketplace subscription.
+- Once the contract is accepted, you'll be prompted to finalize the order by clicking a button. You'll then be directed to the LocalStack Web Application, where you need to authenticate by signing in or signing up.
+    {{< img-simple src="localstack-web-application-contract.png" alt="Image of LocalStack Web Application showing a subscribe button which can be clicked after signing-in to the application">}}
+- Upon successful authentication on the LocalStack Web Application, you'll be automatically returned to the previous page. At this point, an additional button will be visible, allowing you to subscribe to LocalStack.
+    {{< img-simple src="localstack-web-application-subscribe.png" alt="Image of LocalStack Web Application showing a subscribe button which allows you to complete the subscription">}}
+- Click the button to establish a connection between your LocalStack organization and the AWS Marketplace Customer. This will redirect you to the subscription page, where a banner appears next to existing subscriptions, with the AWS Marketplace subscription becoming visible within a few minutes.
+    {{< img-simple src="localstack-web-application-synchronization.png" alt="Image of LocalStack Web Application showing a box which mentions that the synchronization is in progress and may take a few minutes">}}
```

Comment:
Not sure if that works, but basically it would be nice to reduce the size of this image a bit (to align it with the other images):
```suggestion
    {{< img-simple src="localstack-web-application-synchronization.png" width=350 alt="Image of LocalStack Web Application showing a box which mentions that the synchronization is in progress and may take a few minutes">}}
```